use-timescale/security/multi-factor-authentication.md

```diff
@@ -8,33 +8,31 @@ tags: [two-factor user authentication]
 
 # Multi-factor user authentication
 
-You can use two-factor authentication to log in to your Timescale account.
+You can use two-factor authentication to log in to your $COMPANY account.
 Two-factor authentication, also known as two-step verification or 2FA, enables
-secure logins that requires an authentication code in addition to your user
-password. The code is provided by an app on your mobile device, such as Google
-Authenticator.
+secure logins that require an authentication code in addition to your user
+password. The code is provided by an authenticator app on your mobile device. There are multiple authenticator apps available. 
+This page describes how to configure two-factor authentication with Google Authenticator.
 
-## Configure two-factor authentication with Google Authenticator
-
-Google Authenticator is a useful app for keeping track of your two-factor
-authentication keys. However, there are other authenticators available to suit
-your needs.
+## Prerequisites
 
 Before you begin, make sure you have:
 
 *   Installed the [Google Authenticator application][install-google-authenticator]
-  on your mobile device.
+    on your mobile device.
 
-<Procedure>
+## Configure two-factor authentication with Google Authenticator
 
-### Configuring two-factor authentication with Google Authenticator
+Take the following steps to configure two-factor authentication:
 
-1.  [Log in to your Timescale account][cloud-login].
-1.  Click the `User name` icon in the top-right of the console and select `Account`.
+<Procedure>
+
+1.  [Log in][cloud-login] to your $COMPANY account.
```

Comment:
You are logging into Timescale Console to access your account settings. 

Login to [$CONSOLE][cloud-login]. 

Same for all other login stesp.